cpp/cmake/thirdparty/get_arrow.cmake

```
@@ -320,6 +346,7 @@ function(find_and_configure_arrow VERSION BUILD_STATIC EXCLUDE_FROM_ALL ENABLE_P
       rapids_export_package(BUILD Parquet cudf-exports)
       rapids_export_package(BUILD ArrowDataset cudf-exports)
     endif()
+    rapids_export_package(BUILD ArrowCompute cudf-exports)
```

Guessing this may also need to be installed?
```suggestion
    rapids_export_package(BUILD ArrowCompute cudf-exports)
    rapids_export_package(INSTALL ArrowCompute cudf-exports)
```

---

Done